File: 1768-merge-strings-alternately/Solution.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        StringBuilder output = new StringBuilder();
        int length = Math.max(word1.length(),word2.length());

        for(int i = 0; i < length; i++){
            if(i < word1.length()) output.append(word1.charAt(i));
            if(i < word2.length()) output.append(word2.charAt(i));
        }

        return output.toString();
    }
}

File: 1768-merge-strings-alternately/Solution.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function(word1, word2) {
    let output = "";
    let length = Math.max(word1.length, word2.length);

    for (let i = 0; i < length; i++) {
        if (i < word1.length) output += word1[i];
        if (i < word2.length) output += word2[i];
    }

    return output;
};
